src/lib/y2partitioner/widgets/tabs.rb

```diff
@@ -11,7 +11,7 @@ module TabsWithState
       # Overrides default behavior of tabs to register the new state before
       # doing the real switch
       def switch_page(page)
-        UIState.instance.switch_to_tab(page)
+        UIState.instance.switch_to_tab(page.label)
```

Comment:
Why not page.id?

Reply:
Because `page` here is not a `Widgets::Pages::Base` but `Widgets::PartitionsTab` (a `CWM::Tab`). And tabs has not `.id` method.

Reply:
So... 

The method should be `switch_tab` and the param `tab`

Isn't it?